### ISSUE TITLE
Wiki add page button and page sort order

### ISSUE THREAD

**Issue by HaakonME** [redmatrix/hubzilla#719]
I am trying to create a wiki with over 140 articles by copying articles from another wiki, and have discovered three issues in Hubzilla (standard) version 2.3.4 at Parlementum.net.

The Add page button is currently located below the list of new pages, moving it further and further down and out of sight as it gets below the fold. Scrolling is not sufficient, and I have resorted to using CTRL and + and - to zoom in and out to keep going. It would be nice if the Add page button was at the top of the list of pages.

The list of articles in the wiki are neither listed in the order added (by time) nor in alphabetically (by letter). Here is the top of the list as it looks now:

    Home
    De tolv villender
    De tre prinsesser i Hvittenland
    Askeladden som stjal sølvendene til trollet
    En aften ved Andelven
    "Tsju pus, vil du ned av bordet!"
    Den syvende far i huset
    "God dag, mann!" - "Økseskaft"
    Dukken i gresset
    Askeladden som kappåt med trollet
    Det blå båndet
 
The articles were added in alphabetical order from a list in another wiki - http://far.no/index.php?title=Kategori:Eventyr - starting with "God dag, mann!" - "Økseskaft". That site uses Mediawiki.

If the sort order could be fixed to either be alphabetical or in the order added or a way to reorder it, that would be great.

The third is issue that of taxonomy. In Mediawiki it is possible to add categories and subcategories to articles, i.e. a sort of tagging, which is either hierarchical or flat. All the articles linked to above have the category "Eventyr" or "Fairy tales" and Mediawiki automatically sorts articles in a category by letter, so you can easily find those articles starting with for instance an M. Is something similar in the works for the Hubzilla Wiki? If not, it would be a great addition.

**Comment by HaakonME** [comment]
Just noticed: a small adjustment is needed to push down the text in the wiki text field - notice how the top of the h3-tags disappear. 
![image](https://cloud.githubusercontent.com/assets/577857/25091080/70fd63c2-2387-11e7-8da1-4eff9ae198a7.png)

**Comment by git-marijus** [comment]
I just made a commit that will order pages by creation date. 09967598e

**Comment by git-marijus** [comment]
Not sure if we should move the "Add new page" link to the top. It seems logical to be at the bottom. Maybe we can find another solution...

**Comment by HaakonME** [comment]
I really think we need to move the "Add new page"-link to the top, and sort the list of pages by initial letter rather than creation time.
![screenshot-parlementum net-2017-04-22-16-32-15](https://cloud.githubusercontent.com/assets/577857/25305363/424241d2-277a-11e7-88d4-38a08e029a04.png)

**Comment by ilu33** [comment]
I have the same problem as Haakon. Same goes for the "Save" button. I think the button should be at the top AND the bottom. I'm keeping a lot of really long wiki pages and mostly I just replace the existing text by pasting in the new version. This leaves me at the top of the page without a button. The amount of scrolling I have to do is annoying.

**Comment by git-marijus** [comment]
@anaqreon is there a specific reason why we don't save the wiki page title in item_title?

**Comment by anaqreon** [comment]
No. When I originally created the git based wiki the title was equivalent to the file name, and I didn't use the Hubzilla database to store that information at all if I recall correctly.

**Comment by git-marijus** [comment]
Maybe @zotlabs had a specific reason?

**Comment by zotlabs** [comment]
No specific reason. It started as a filename. I wasn't sure if I was going to run into problems putting it in the title field because the code wasn't completely familiar to me - so I kept it separate just in case the distinction turned out to be important.

**Comment by git-marijus** [comment]
It makes sense to sort the pages alphabetically to me. This will be only possible if we save the page title with the item i guess. Looking for the titles in iconfig and than sort them seems not so efficient...

**Comment by anaqreon** [comment]
Moved the add page button to top of the wiki page list: 6239a27
